README.md

```diff
@@ -85,8 +85,8 @@ Due to the differences in execution environments between ShellJS and `shx` (JS v
 | Unsupported command | Recommend workaround |
 | ------------------- | -------------------- |
 | `shx cd`            | Just use plain old `cd` (it's the same on windows too) |
-| `shx pushd`         | No workaround |
-| `shx popd`          | No workaround |
+| `shx pushd`         | Just use plain old `pushd`. Use forward slashes and double-quote the path. (e.g. `pushd "../docs"`. This would fail on Windows without the quotes) |
```

Comment:
Most cmd commands and all cmd examples expect backward slashes (like all of Windows). Some commands however support forward slashes, and so do the lower level win32 API functions. Other cmd commands, however, choke on forward slashes and I don't know why. The only explanation I have is this:

![image](https://user-images.githubusercontent.com/703546/29495196-15f54f40-85ba-11e7-9eae-316924f85b51.png)